client-apis/encrypt.md

```diff
@@ -113,68 +171,185 @@ This document refers to the output of the key derivation algorithm as the derive
 Note that if the key derivation algorithm is the [identity KDF](../framework/algorithm-suites.md#identity-kdf),
 then the derived data key is the same as the plaintext data key.
 
-### Message
+The frame length used in the procedures described below is the input [frame length](#frame-size),
+if supplied.
+If a frame length is not specified on input, this operation MUST choose a reasonable value to
+default to.
```

Comment:
If possible we should define what a "reasonable value" is (or at least how to determine one).